c7n/policy.py

```diff
@@ -638,6 +657,24 @@ def provider_name(self):
     def region(self):
         return self.data.get('region')
 
+    @property
+    def tz(self):
+        if 'tz' in self.data:
+            return pytz.timezone(self.data.get('tz'))
+        return pytz.utc
+
+    @property
+    def start(self):
+        if self.data.get('start'):
+            return parse(self.data.get('start'))
```

Comment:
these should get returned localized with tz instead of forcing callers to localize themselves.